pkg/frontend/admin_hive_syncset_resources.go

```diff
@@ -0,0 +1,65 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"net/http"
+	"path/filepath"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"github.com/ugorji/go/codec"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/frontend/middleware"
+)
+
+func (f *frontend) getAdminHiveSyncsetResources(w http.ResponseWriter, r *http.Request) {
```

Comment:
Maybe rename the function to getAdminHiveClusterSync so it reflect accurately as we are getting clustersync object and not syncset resources.

Reply:
updated